### ISSUE TITLE
USB input device is passthrough-ed to the container if plugged in while Android Session is active

### ISSUE THREAD

**Issue by RushingAlien** [waydroid/waydroid#778]
**Describe the bug**
When a USB input device is plugged in while android session is active, it seems to be passthrough-ed? as it asks for device setup and there will be dual cursors, one from the host, one from the container

**Screenshots**
[Screencast dari 2023-02-24 07-29-29.webm](https://user-images.githubusercontent.com/80886145/221062439-02740418-31cb-4858-a634-3a94a1aee6d8.webm)

**General information (please complete the following information):**
 - Waydroid tools 1.4.0
 - Waydroid Images 20230218

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - GPU: AMD Ryzen™ 5 3500U with Radeon™ Vega Mobile Gfx × 8
 - Kernel version: 6.1.12-zen1-1-zen
 - Host mesa version: 22.3.6
 - Desktop: GNOME 43.3

**Additional context**

**Logs (please upload as file)**
 - Prop file, located at `/var/lib/waydroid/waydroid.prop`. In case that doesn't exist, use `/var/lib/waydroid/waydroid_base.prop` instead
 - Log file, located at `/var/lib/waydroid/waydroid.log`
 - *Full* (not partial) output of `waydroid logcat` while Waydroid is running
 - Output of `dmesg`
[waydroid.prop.txt](https://github.com/waydroid/waydroid/files/10819614/waydroid.prop.txt)
[waydroid.log](https://github.com/waydroid/waydroid/files/10819608/waydroid.log)
[waydroid logcat.txt](https://github.com/waydroid/waydroid/files/10819604/waydroid.logcat.txt)
[dmesg.txt](https://github.com/waydroid/waydroid/files/10819612/dmesg.txt)

**Comment by magbulogtong81** [comment]
i actually got the same issue (#762), but after 2 weeks there's still no reply

**Comment by dthusian** [comment]
You can work around this issue with `waydroid prop set persist.waydroid.uevent false`. This prevents all USB devices from being passed through.

**Comment by selurvedu** [comment]
> You can work around this issue with `waydroid prop set persist.waydroid.uevent false`. This prevents all USB devices from being passed through.

I'd like to pass through my gamepad, but not my mouse. Is that possible?

**Comment by whale2186** [comment]
> > You can work around this issue with `waydroid prop set persist.waydroid.uevent false`. This prevents all USB devices from being passed through.
> 
> I'd like to pass through my gamepad, but not my mouse. Is that possible?
> 
> Ref: #762 #135

did you fix this issue ?

**Comment by selurvedu** [comment]
@whale2186 no. This means I have to avoid replugging the mouse while Waydroid is active.
